### PR TITLE
[setupapi]: Fix a FIXME in SetupDiSetDeviceRegistryPropertyA.

### PR DESCRIPTION
**Pull request by andy-123**
Rename SetupDiSetDeviceRegistryPropertyW to IntSetupDiSetDeviceRegistryPropertyAW and use it for both (A/W).

The FIXME in the A-Version was already implemented in the W-Version.

CORE-14173

### CODE REVIEW

**Comment by ThFabba** [comment]
The usual method is just to convert ansi strings to wide strings, call the W function, then convert wide strings back to ansi strings. Is that not feasible here?

**Comment by ThFabba** [comment]
Not sure what patch you're looking at @Fraizeraust, his real name and email are right there.

**Comment by Fraizeraust** [comment]
@ThFabba Correct me if I am wrong but is Andreas Mohr actual his real name? Because that seems confusing to me.

**Comment by ThFabba** [comment]
They're not the same person just because they touch the same file. :p
Andreas Mohr used to work for CodeWeavers, andy-123 is a known ROS contributor.
https://source.winehq.org/git/wine.git/?a=search&h=HEAD&st=commit&s=Andreas+Mohr
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=commit&s=Andreas+Maier

**Comment by andy-123** [comment]
I'm Andreas Maier.

File: dll/win32/setupapi/devinst.c [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * SetupAPI device installer
  *
  * Copyright 2000 Andreas Mohr for CodeWeavers
- *           2005-2006 Herv� Poussineau (hpoussin@reactos.org)
+ *           2005-2006 Hervé Poussineau (hpoussin@reactos.org)
```
Comment (binarymaster) [inline dll/win32/setupapi/devinst.c]: Whitespace change.

Comment (andy-123) [reply dll/win32/setupapi/devinst.c]: I can't fix it, sorry. For me it seems there is no difference between these two lines.

Comment (gigaherz) [reply dll/win32/setupapi/devinst.c]: It's not a whitespace change, it's an encoding change:

firefox_2018-01-04_16-05-47

Comment (HBelusca) [reply dll/win32/setupapi/devinst.c]: It seems the new encoding is UTF-8, which is perfectly OK.

Comment (binarymaster) [reply dll/win32/setupapi/devinst.c]: image

Interesting... for me it looks like this. Maybe browser use some workaround?

Comment (gigaherz) [reply dll/win32/setupapi/devinst.c]: It looks like that in github, that's why I pasted a screenshot of the email.

Comment (andy-123) [reply dll/win32/setupapi/devinst.c]: Ahrg, changed it now to WIN-1252 (last commit "Whitspace fix") ... @HBelusca . I've readed your comment after that.

**Comment by andy-123** [comment]
Sorry. I tried to change the wrong character (in comment) directly in github-gui. Now I have set my real name in github settings. I think i never need to change it directly.
Can i remove these two commits? They aren't really needed.

**Comment by HBelusca** [comment]
@andy-123 : Correctly fix your user name and email in your local work copy, squash all of these commits into one, then (possibly) amend the resulting commit to fix any remaining thing (username + mail), then push --force-with-lease to your remote branch. The PR should update itself and have everything correct.

**Comment by andy-123** [comment]
Done. 
@HBelusca Thank you for your help.

**Comment by HBelusca** [comment]
Good :) Just a comment: it seems that your commit is however not "associated" to your github account, is it normal?

**Comment by andy-123** [comment]
You're right. I used a different email. Thank you.

**Pull request merged**